requirements.txt.in

```diff
@@ -20,4 +20,4 @@ urllib3 >= 1.21.1
 requests-http-signature >= 0.0.3
 aws-xray-sdk >= 1.0
 pyjwt >= 1.6.4
-pyyaml >= 4.2b1
+pyyaml >= 4.2b1, <= 5.1.0
```

Comment:
Should we make this `5.1` or `5.2`, so that if/when the aws-cli is updated to bump their YAML support we don't have to make another change? Or, would that lead to the requirement of PyYAML 5.1.x resolving to 5.1.1 and ending up in the same place we were before?

Reply:
We'd end up in the same place we were before.